deployments/liqo_chart/crds/dashboard.liqo.io_views.yaml

```diff
@@ -0,0 +1,67 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
I suppose this is probably auto-generated, but `apiextensions.k8s.io/v1beta1` is deprecated in v1.16 in favor of `apiextensions.k8s.io/v1` (just for information, I believe this is currently not a problem)